src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2991,28 +3036,26 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 			'qr_ticket_id'  => $attendee->ID,
 			'security_code' => $security,
 
-			// Attendee Meta
+			// Attendee Meta.
 			'attendee_meta' => $meta,
 
 			// Handle initial Attendee flags.
-			// @todo Make these live in future IAC work.
-			'is_subscribed' => false,
+			'is_subscribed' => tribe_is_truthy( get_post_meta( $attendee->ID, $this->attendee_subscribed, true ) ),
 			'is_purchaser'  => true,
-		) );
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 7e9fad121c2b2c04089e71a3bcc934f7</sub>